cloud: add storage usage impact notes for TiKV and TiFlash (#22514)
pingcap/docs: tidb-cloud/monitor-new-relic-integration.md

| tidb_cloud.db_command_per_second| gauge | type: Query\|StmtPrepare\|...<br/><br/>cluster_name: `<cluster name>`<br/><br/>instance: tidb-0\|tidb-1…<br/><br/>component: `tidb` | The number of commands processed by TiDB per second, which is classified according to the success or failure of command execution results. |
| tidb_cloud.db_queries_using_plan_cache_ops| gauge | cluster_name: `<cluster name>`<br/><br/>instance: tidb-0\|tidb-1…<br/><br/>component: `tidb` | The statistics of queries using [Plan Cache](/sql-prepared-plan-cache.md) per second. The execution plan cache only supports the prepared statement command. |
| tidb_cloud.db_transaction_per_second| gauge | txn_mode: pessimistic\|optimistic<br/><br/>type: abort\|commit\|...<br/><br/>cluster_name: `<cluster name>`<br/><br/>instance: tidb-0\|tidb-1…<br/><br/>component: `tidb` | The number of transactions executed per second. |

> | tidb_cloud.node_storage_used_bytes | gauge | cluster_name: `<cluster name>`<br/><br/>instance: tikv-0\|tikv-1…\|tiflash-0\|tiflash-1…<br/><br/>component: tikv\|tiflash | The disk usage, in bytes, for TiKV or TiFlash nodes. This metric primarily represents the logical data size in the storage engine, and excludes filesystem overhead, WAL files, and temporary files. To calculate the actual disk usage rate, use `(capacity - available) / capacity` instead. When usage of TiKV exceeds 80%, latency might increase, and higher usage might cause requests to fail. Usage of Tiflash above 80% indicates disk exhaustion and requests might fail. |

Reviewer: ```suggestion
| tidb_cloud.node_storage_used_bytes | gauge | cluster_name: `<cluster name>`<br/><br/>instance: tikv-0\|tikv-1…\|tiflash-0\|tiflash-1…<br/><br/>component: tikv\|tiflash | The disk usage, in bytes, for TiKV or TiFlash nodes. This metric primarily represents the logical data size in the storage engine, and excludes filesystem overhead, WAL files, and temporary files. To calculate the actual disk usage rate, use `(capacity - available) / capacity` instead. Storage usage of TiKV exceeding 80% can cause latency spikes, and higher usage might cause requests to fail. Storage usage of TiFlash exceeding 80% indicates disk exhaustion and requests might fail. |
```

> | tidb_cloud.node_storage_used_bytes | gauge | cluster_name: `<cluster name>`<br/><br/>instance: tikv-0\|tikv-1…\|tiflash-0\|tiflash-1…<br/><br/>component: tikv\|tiflash | The disk usage, in bytes, for TiKV or TiFlash nodes. This metric primarily represents the logical data size in the storage engine, and excludes filesystem overhead, WAL files, and temporary files. To calculate the actual disk usage rate, use `(capacity - available) / capacity` instead. Storage usage of TiKV exceeding 80% can cause latency spikes, and higher usage might cause requests to fail. When the storage usage of all TiFlash nodes reaches 80%, executing a DDL statement to add a TiFlash replica hangs indefinitely.|

Reviewer: ```suggestion
| tidb_cloud.node_storage_used_bytes | gauge | cluster_name: `<cluster name>`<br/><br/>instance: tikv-0\|tikv-1…\|tiflash-0\|tiflash-1…<br/><br/>component: tikv\|tiflash | The disk usage, in bytes, for TiKV or TiFlash nodes. This metric primarily represents the logical data size in the storage engine, and excludes filesystem overhead, WAL files, and temporary files. To calculate the actual disk usage rate, use `(capacity - available) / capacity` instead. The storage usage of TiKV exceeding 80% can cause latency spikes, and higher usage might cause requests to fail. When the storage usage of all TiFlash nodes reaches 80%, executing a DDL statement to add a TiFlash replica hangs indefinitely.|
```
